### DISCUSSION TITLE
How schedule the 'Accepted Booking Reminder Email'

### DISCUSSION THREAD

**Discussion by f14m07** [liedekef/events-made-easy#643] [Q&A]
Hi, 
Is it possible to schedule the reminder e-mail to be sent to bookers and choose the deadline? Maybe with a hook?

**Comment by liedekef** [comment]
Do you mean this:
![image](https://github.com/user-attachments/assets/372683cb-b357-4357-9b67-f31da3993769)

**Reply by f14m07** [reply]
Yes, is it possible to schedule this in hours?

**Reply by liedekef** [reply]
in hours? No, currently not. The logic is made so that it is only sent once a day.

**Comment by f14m07** [comment]
Is it schedule by the cron/hook eme_cron_event_daily_actions? If this is the case, I could set the hour?

**Comment by liedekef** [comment]
That schedule is meant to be executed once daily, but it doesn't really matter when.
